docs/api/datasets/non_geo_datasets.csv

```diff
@@ -3,6 +3,7 @@ Dataset,Task,Source,License,# Samples,# Classes,Size (px),Resolution (m),Bands
 `Benin Cashew Plantations`_,S,Airbus Pléiades,"CC-BY-4.0",70,6,"1,122x1,186",10,MSI
 `BigEarthNet`_,C,Sentinel-1/2,"CDLA-Permissive-1.0","590,326",19--43,120x120,10,"SAR, MSI"
 `BioMassters`_,R,Sentinel-1/2 and Lidar,"CC-BY-4.0",,,256x256, 10, "SAR, MSI"
+`BRIGHT`_,CD,"MAXAR, NAIP, Capella, Umbra","CC-BY-NC-4.0",3239,4,"0.1-1","RGB,SAR"
```

Comment:
```suggestion
`BRIGHT`_,CD,"MAXAR, NAIP, Capella, Umbra","CC-BY-4.0 AND CC-BY-NC-4.0",3239,4,"0.1-1","RGB,SAR"
```
Technically these are probably equivalent, but someone may want to use only the images or only the masks.

Reply:
one is Non-commerrcial and the other is right? That's quiet a big difference. And the HF says that the vast majority of the data is Non-commercial since it's from Maxar, so I thought safer to put NC

Reply:
`CC-BY-4.0 AND CC-BY-NC-4.0` implies you need to satisfy both licenses, see https://spdx.dev/learn/handling-license-info/

Reply:
cool, learned something again, thanks!